staging/src/k8s.io/code-generator/cmd/validation-gen/validators/openapi.go

```diff
@@ -66,8 +67,12 @@ func (openAPIDeclarativeValidator) ExtractValidations(t *types.Type, comments []
 }
 
 func FormatValidationFunction(format string) FunctionGen {
+	//FIXME: Let's define the naming convention for these - jsonStyle or GoStyle?
```

Comment:
For OpenAPI compatibility of format names I believe we want `lower-case`. https://json-schema.org/draft/2020-12/json-schema-validation#name-defined-formats

Reply:
that suggests "dns-1123-label" or "dns1123-label" as a convention?